src/data/ua_overrides.js

```diff
@@ -677,6 +677,42 @@ const AVAILABLE_UA_OVERRIDES = [
       },
     },
   },
+  {
+    /*
+     * Bug 1673937 - Add UA override for goal.com
+     *
+     * This site needs to have Chrome into its UA string to be able
+     * to serve the right experience on both desktop and mobile.
+     */
+    id: "bug1673937",
+    platform: "all",
+    domain: "goal.com",
+    bug: "1673937",
+    config: {
+      matches: ["*://goal.com/*"],
+      uaTransformer: originalUA => {
+        return originalUA + " Chrome/86.0.4240.110";
+      },
+    },
+  },
+  {
+    /*
+     * Bug 1671150 - Change UA for nha.chotot.com to not be Firefox but anything else
+     *
+     * Any mention of "Firefox" causes this degraded site, but everything else
+     * is fine, such as WebcompatFox
+     */
+    id: "bug1671150",
+    platform: "all",
+    domain: "nha.chotot.com",
+    bug: "1671150",
+    config: {
+      matches: ["*://nha.chotot.com/*"],
+      uaTransformer: originalUA => {
+        return originalUA.replace("Firefox", "WebcompatFox");
+      },
+    },
+  },
```

Comment:
We don't need this anymore, the site luckily fixed it! 🥳

Reply:
fixed